docs/devops/getting-started.md

```diff
@@ -22,18 +22,28 @@ image:
 ## Best DevOps practices at SAP ##
 
 Continuous delivery is a method to develop software with short feedback cycles.
-It is applicable to projects both for SAP Cloud Platform and SAP on-premise platforms.
-SAP implements tooling for continuous delivery in project "Piper".
-The goal of project "Piper" is to substantially ease setting up continuous delivery in your project using SAP technologies.
+It applies to projects both for SAP Cloud Platform and SAP on-premise platforms.
+SAP implements tooling for continuous delivery in the project "Piper".
+The goal of the project "Piper" is to substantially ease setting up continuous delivery in your project using SAP technologies.
 
 ### SAP Cloud SDK for Continuous Delivery ###
 
 [SAP Cloud SDK for Continuous Delivery](https://github.com/SAP/cloud-s4-sdk-pipeline) is open source and free to use.
-This ready-made pipeline is part of project "Piper" and allows you to start with continuous delivery without writing any pipeline code -- only declarative configuration is required.
-It helps you to quickly deliver high quality applications to SAP Cloud Platform.
+This ready-made pipeline is part of the project "Piper" and allows you to start with continuous delivery without writing any pipeline code -- only declarative configuration is required.
+It helps you to quickly deliver high-quality applications to SAP Cloud Platform.
 
 Check out the [documentation of SAP Cloud SDK for Continuous Delivery](https://sap.github.io/jenkins-library/pipelines/cloud-sdk/introduction/) for more details and usage hints.
 
-### Project "Piper" ###
+## Where to look next? ##
 
-For more information on continuous integration and continuous delivery (CI/CD) in the SAP ecosytem, please refer to the documentation of [project "Piper"](https://sap.github.io/jenkins-library).
+### Learn about project "Piper" ###
+
+For more information on continuous integration and continuous delivery (CI/CD) in the SAP ecosystem, please refer to the documentation of [project "Piper"](https://sap.github.io/jenkins-library).
+
+### Open SAP courses ###
+Check **Week 3** of [Create and Deliver Cloud-Native SAP S/4HANA Extensions](https://open.sap.com/courses/s4h13/items/5F9Yrn3QwX0aIFg7JmSS9H) course.
```

Comment:
caution, this is about 2,5 years old, not sure if we should still promote it

Reply:
What should we promote instead? Some good tutorial or Just Piper website is enough?

Reply:
I'm not sure if it does more good than harm, that's all I've wanted to say. We could reference Daniel's session from the CAP course, but this was only one unit, so it has less content of course. We don't have an updated replacement for the course, so it might also be ok to link it, but I guess it might cause some confusion due to its age.

Reply:
@fwilhe, I actually agree here, I thought if it's not taken down from Open SAP it's still relevant. I'll remove it then. I thought we have some nice content. Do we plan to refresh that course offering?

Reply:
@sacnl, do you think we need anything else on Continuous Delivery, or let's keep it concise?

Reply:
> Do we plan to refresh that course offering?

I don't think we will do anytime soon, it was a huge project.

Reply:
> need anything else on Continuous Delivery, or let's keep it concise

I know you asked Henning, but imo we can merge the PR and iterate on it later. I guess this is also what Henning would have said :)

Reply:
I think most of the concept level content from that S4H13 course is still valid to get an introduction. Add to that Daniel's recent session, and this gives a good starting point.

Reply:
Good guys, I'll merge it and keep looking for good ideas for Continous Delivery.